integrations/snyk/snyk/client.py

```diff
@@ -196,7 +196,8 @@ async def get_paginated_targets(
             logger.info(f"Fetching paginated targets for organization: {org['id']}")
 
             url = f"/orgs/{org['id']}/targets"
-            query_params = {"version": f"{self.snyk_api_version}~beta"}
+            target_api_version = "2024-05-23"
```

Comment:
This should be held next to the other version

Reply:
I didn't quite get this

Reply:
To be in the self properties of the class, same as the other snyk version that you are maintaining.